mos_tests/neutron/python_tests/test_dvr.py

```diff
@@ -253,3 +259,344 @@ def test_routing_after_reset_computes(self, prepare_openstack, env_name):
         self.check_ping_from_vm(vm=self.server2,
                                 vm_keypair=self.instance_keypair,
                                 ip_to_ping=self.server1_ip)
+
+    def try_to_update_router_to_centralized(self, router_id):
```

Comment:
This is check, so let's call it like `check_exception_on_router_update_to_centralize`